### PR TITLE
Updated image_key documentation

### PR DESCRIPTION
**Pull request by sanjanag**
Screenshot 2024-03-08 at 12 01 50 PM
Screenshot 2024-03-08 at 12 04 13 PM

### CODE REVIEW

File: cleanlab/datalab/datalab.py
Comment (jwmueller) [inline cleanlab/datalab/datalab.py:97]:
```suggestion
        Optional key that can be specified for image datasets to point to the field (column) containing the actual images themselves (as PIL objects).
        If specified, additional image-specific issue types will be checked for in the dataset.
        See the `CleanVision package <https://github.com/cleanlab/cleanvision?tab=readme-ov-file#clean-your-data-for-better-computer-vision>`_ for descriptions of these image-specific issue types.
        Currently, this argument is only supported for data formatted as a Hugging Face ``datasets.Dataset`` object.
```

File: docs/source/cleanlab/datalab/guide/issue_type_description.rst
Comment (jwmueller) [inline docs/source/cleanlab/datalab/guide/issue_type_description.rst:162]:
```suggestion
This functionality currently works only with Hugging Face datasets. You can convert other local dataset formats into a Hugging Face dataset by following `this guide <https://huggingface.co/docs/datasets/en/loading>`_.
```

**Review by jwmueller — APPROVED** [review]
had a couple suggestions but otherwise looks good to merge

**Review by elisno — APPROVED** [review]
LGTM!
Don't merge this until these have been merged, in the following order:

1. #1051 
2. #1052

**Pull request merged**
